<launch>    

  <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform_node" respawn="true" output="screen">
    <rosparam command="load" file="$(find bolt_planner)/params/navsat_transform.yaml" />
    
    <remap from="/odometry/filtered" to="/odom"/>
    <remap from="/gps/fix" to="/gps_waypoints/navsat"/>
    <!--<remap from="imu/data" to="/zed2i/zed_node/imu/data"/>-->
    <remap from="/odometry/gps" to="/gps_waypoints/odom"/>
  </node>

  <node name="gps_waypoints" pkg="bolt_planner" type="gps_waypoints.py" output="screen"/>

</launch>